From 5241e4d7b177d0b6f073cfc9ed5444bf51ec89d6 Mon Sep 17 00:00:00 2001
From: Quentin Armitage <quentin@armitage.org.uk>
Date: Wed, 31 Oct 2018 21:57:02 +0000
Subject: [PATCH] Fix compile warning introduced in commit c6247a9

Commit c6247a9 - "Add command line and configuration option to set umask"
introduced a compile warning, although the code would have worked OK.

Signed-off-by: Quentin Armitage <quentin@armitage.org.uk>
---
 keepalived/core/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/keepalived/core/main.c b/keepalived/core/main.c
index 6670ada28..03757f6f6 100644
--- a/keepalived/core/main.c
+++ b/keepalived/core/main.c
@@ -882,7 +882,7 @@ set_umask(const char *optarg)
 
 	if (*endptr || umask_long < 0 || umask_long & ~0777L) {
 		fprintf(stderr, "Invalid --umask option %s", optarg);
-		return;
+		return 0;
 	}
 
 	umask_val = umask_long & 0777;
